src/priority.c

```diff
@@ -0,0 +1,113 @@
+// SPDX-License-Identifier: Apache-2.0
+
+/*
+ * Copyright 2022 Joel E. Anderson
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <ctype.h>
```

Comment:
the header check utility will complain about this because it doesn't know about ctype-provided symbols. You'll need to add `isdigit` and `toupper` to the `tools/check_headers/standard_library.yml` file to resolve the static check failure.

Reply:
Thanks, I didn't know how to fix this failure.
Done